sort.php

```diff
@@ -37,7 +37,16 @@ function bubbleSort($sort_array)
     return $sort_array;
 }
 
-if($argc > 1 && substr($argv[1], 0, 1) !== '-') {
+$shortOpts = "n:";
```

Comment:
Please explain what's the `:` for.

Reply:
This means that a value is required after the input n option

Reply:
Please give an example value you'd pass to -n.

Reply:
In our case like this ``php sort.php -n in.txt``

Reply:
Yes, you run the program like this, but -n and in.txt are independent, so in.txt **is not** the value for -n, i.e. -n has no value. Please remove the `:`.